spec/dummy/config/initializers/waste_exemptions_engine.rb

```diff
@@ -20,4 +20,8 @@
 
   # Last Email Cache config
   config.use_last_email_cache = ENV["USE_LAST_EMAIL_CACHE"] || "false"
+
+  # Renewing config
+  config.renewal_window_open_before_days = ENV["RENEWAL_WINDOW_OPEN_BEFORE_DAYS"] || 28
```

Comment:
My only quibble is around the env var names, which then has had a knock on effect to the config and other names.

They are both doing the same thing, holding the number of days for a named period of time. However we seem to have ended up with a different naming convention for both.

My preference would be

```suggestion
  config.registration_renewal_window = ENV["REGISTRATION_RENEWAL_WINDOW"] || 28
  config.registration_grace_window = ENV["REGISTRATION_GRACE_WINDOW"] || 30
```

I've never been that bothered about stating it's in days, but I know @cintamani has had other thoughts on this. But my main aim is to get the names consistent. So as long as that is achieved I'll be happy 😁

Reply:
Eheh :D I wanted to make sure the ENV variable contains an integer that express Days, so I am keep to keep the `-days` stated somewhere for documentation purposes. Just as good practice that any variable states what kind of value it holds, really :)

Reply:
How do we all feel about `renewal_window_before_expiry_in_days` and `renewal_window_after_expiry_in_days`? Feels like the most explicit about what it is.